solr/core/src/java/org/apache/solr/response/transform/RawValueTransformerFactory.java

```diff
@@ -79,7 +84,18 @@ public DocTransformer create(String display, SolrParams params, SolrQueryRequest
     }
 
     if(apply) {
-      return new RawTransformer( field, display );
+      boolean indent = req.getParams().getBool("indent", false);
+      ObjectMapper mapper = null;
+      if (indent) {
+        if (applyToWT.equalsIgnoreCase("json")) {
+          mapper = new ObjectMapper();
+        }
+        else {
```

Comment:
There's a log more WT's possible? https://lucene.apache.org/solr/guide/7_6/response-writers.html#response-writers probably don't want to format all of then as xml, Although I admit I haven't yet figured out when applyToWT gets set, this else makes me nervous. Use of a switch would seem better, especially if we later want to indent some other format (hopefully someday wt=yaml...). Also what about wt=geojson? that's json too. (not sure if the existing code handles :[geojson] though...)